engine/action_contact.go

```diff
@@ -163,6 +163,62 @@ func (c *Client) UpdateContact(ctx context.Context, id, fullName string, metadat
 	return contact, nil
 }
 
+// AdminCreateContact creates a new contact - xpubId is retrieved by the creatorPaymail.
+func (c *Client) AdminCreateContact(ctx context.Context, contactPaymail, creatorPaymail, fullName string, metadata *Metadata) (*Contact, error) {
+	creatorPaymailAddr, err := getPaymailAddress(ctx, creatorPaymail, c.DefaultModelOptions()...)
+	if err != nil {
+		return nil, spverrors.ErrCouldNotFindPaymail.Wrap(err)
+	}
+	if creatorPaymailAddr == nil {
+		return nil, spverrors.ErrCouldNotFindPaymail
+	}
+
+	creatorXPub, err := getXpubByID(ctx, creatorPaymailAddr.XpubID, c.DefaultModelOptions()...)
+	if err != nil {
+		return nil, spverrors.ErrCouldNotFindXpub.Wrap(err)
+	}
+
+	newContactSanitisedPaymail, err := c.PaymailService().GetSanitizedPaymail(contactPaymail)
```

Comment:
```suggestion
	sanitizedPaymail, err := c.PaymailService().GetSanitizedPaymail(contactPaymail)
```
does not sound better ?

Reply:
I had problem with this name, because we have two paymails in a scope of this function:
- one named `contactPaymail`,
- and second named `creatorPaymail`.

With this name i wanted to highlight which paymail is kept in this sanitizedPaymail variable.